gulpfile.js

```diff
@@ -41,40 +44,34 @@ var configLocal = require('./gulp-config.json'),
 // Installation of components/dependencies
 //
 
-// Bower
-gulp.task('bower', function() {
-  return bower.commands.install()
-    .pipe(gulp.dest(config.bowerPath));
-});
-
 // Web font processing
-gulp.task('fonts', ['bower'], function() {
+gulp.task('move-components-fonts', function() {
   // TODO add custom fonts from components?
-  // gulp.src(config.bowerPath + '/path/to/component/fonts/*/*')
+  // CREATE SEPARATE TASK THAT RETURNS A STREAM:
+  // return gulp.src(config.packagesPath + '/path/to/component/fonts/*/*')
   //   .pipe(gulp.dest(config.dist.fontPath));
 
   // TODO add custom fonts from src directory?
-  // gulp.src(config.src.fontPath + '/path/to/fonts/*')
+  // CREATE SEPARATE TASK THAT RETURNS A STREAM:
+  // return gulp.src(config.src.fontPath + '/path/to/fonts/*')
   //   .pipe(gulp.dest(config.dist.fontPath + '/font-name'));
   return;
 });
 
-gulp.task('bootstrap', ['bower'], function() {
-  gulp.src(config.bootstrap.scss + '/**/*', {base: config.bootstrap.scss})
+// Copy Bootstrap scss files
```

Comment:
As long as we're supporting a Bower install of our framework, we have to copy the files. If we decide to go with an npm only path, then we would need to reorganize a little bit to make sure the project builds whenever someone includes it. So we really need to decide if we want to support a Bower install.